src/users/users.controller.ts

```diff
@@ -283,6 +285,42 @@ export class UsersController {
     }
   }
 
+  @Get('/one/:email/isadmin')
+  @UseGuards(AuthGuard)
+  @JwtType(JwtProcessorType.RSA)
+  @ApiOperation({
+    description: SWAGGER_DESC_ADMIN_RIGHTS,
+  })
+  @ApiForbiddenResponse({
+    description: 'user has no admin rights',
+    schema: {
+      type: 'object',
+      properties: {
+        statusCode: { type: 'number' },
+        message: { type: 'string' },
+      },
+    },
+  })
+  @ApiOkResponse({
+    description: 'Returns true if user has admin rights',
+  })
+  async getAdminStatus(@Param('email') email: string) {
+    try {
+      return await this.usersService.findByEmail(email).then(({ isAdmin }) => {
```

Comment:
Avoid mixing `async/await` statements and Promise API. The preferred way is `async/await`